### PR TITLE
Update ug to remove mentions of TAG field

### PR DESCRIPTION
**Pull request by brelkh**
Some other minor edits were made to improve readability.

### CODE REVIEW

**Review by Benjykoh — COMMENTED** [review]
Thanks for updating the UG! Almost LGTM, but I think for some of the add command examples,                                maybe we should include that the module code field is compulsory? Because I am afraid it might cause some confusion for the readers. 🤔

**Comment by brelkh** [comment]
> Thanks for updating the UG! Almost LGTM, but I think for some of the add command examples, maybe we should include that the module code field is compulsory? Because I am afraid it might cause some confusion for the readers. 🤔

@Benjykoh  Thanks for the review! I agree that it might confuse the reader but one other consideration would be that the tips would be more difficult to understand if we include too much information. 

Hmm, maybe if I replace `add` with `edit` command as an example, that would be easier to understand as well as represent our intended implementation of the command?

**Comment by brelkh** [comment]
@Benjykoh I've replaced the `add` command with `edit` so that we can use less fields in the examples given (less confusing for reader), while ensuring that the examples match our actual implementation of those commands. I think it is clearer now but do you think this works better?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -48,25 +49,27 @@ who are teaching multiple classes/modules at the same time. It is optimised for
...
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+* Parameters in square brackets are optional parameters.<br>
+  Example: For the command `edit STUDENT_INDEX [p/PHONE_NUMBER] [m/MODULE_CODE]`, the user must supply the `STUDENT_INDEX` parameter in the input, whereas the `PHONE_NUMBER` and `MODULE_CODE` parameters are optional.<br>
+  Thus, the user can input the command `edit 10 p/98765432` (`MODULE_CODE` not specified) or `edit 10 p/98765432 m/CS2103` (`MODULE_CODE` specified).
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Parameters with `…`​ after them can be used multiple times.<br>
+  Example: For the command `delete STUDENT_INDEX…​`, the user can input `delete 10` (1 `STUDENT_INDEX` parameter) or `delete 10 11 12 13` (multiple `STUDENT_INDEX` parameters).
```
Comment (medajet) [inline docs/UserGuide.md:63]: Would suggest changing "1" to "one" in this case to avoid confusion between the parameter itself and the instruction.

Comment (medajet) [inline docs/UserGuide.md:63]:
```suggestion
  Example: For the command `delete STUDENT_INDEX…​`, the user can input `delete 10` (one `STUDENT_INDEX` parameter) or `delete 10 11 12 13` (multiple `STUDENT_INDEX` parameters).
```

**Review by medajet — APPROVED** [review]
IMO the updates are clear enough, LGTM! Just an extremely minor change for your consideration 👍

File: docs/UserGuide.md (hunk 1 above)
Comment (brelkh) [reply docs/UserGuide.md]: @medajet Thanks for the review! Your suggestion adds clarity for the reader and I've committed it 👍

**Pull request merged**